### ISSUE TITLE
Name of project is shown as CLUTTER_IM_MODULE=ibus when the command is mkproject -t<CR>

### ISSUE THREAD

**Issue by artemisfowl** [artemisfowl/mkproject#1]
When the mkproject program is called out without the necessary details for the name of the project or the type of the project the following output is shown as of now.
![screenshot from 2018-12-15 23-35-24](https://user-images.githubusercontent.com/1042956/50046020-941b8e00-00c2-11e9-9db1-a7c5929abbc4.png)

```bash
$ ./mkproject -t c dummy
Name of the project : dummy
$ ./mkproject -t
Name of the project : CLUTTER_IM_MODULE=ibus
```

**Comment by artemisfowl** [comment]
Now the output seems to be different - it is failing when the argument is not provided.

```bash
$ valgrind --leak-check=full ./mkproject -t
==12287== Memcheck, a memory error detector
==12287== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==12287== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==12287== Command: ./mkproject -t
==12287== 
==12287== Invalid read of size 1
==12287==    at 0x4EE5E70: __strcmp_sse2_unaligned (strcmp-sse2-unaligned.S:24)
==12287==    by 0x108ABE: p_assign_ptype (project.c:98)
==12287==    by 0x10885D: main (main.c:32)
==12287==  Address 0x0 is not stack'd, malloc'd or (recently) free'd
==12287== 
==12287== 
==12287== Process terminating with default action of signal 11 (SIGSEGV)
==12287==  Access not within mapped region at address 0x0
==12287==    at 0x4EE5E70: __strcmp_sse2_unaligned (strcmp-sse2-unaligned.S:24)
==12287==    by 0x108ABE: p_assign_ptype (project.c:98)
==12287==    by 0x10885D: main (main.c:32)
==12287==  If you believe this happened as a result of a stack
==12287==  overflow in your program's main thread (unlikely but
==12287==  possible), you can try to increase the size of the
==12287==  main thread stack using the --main-stacksize= flag.
==12287==  The main thread stack size used in this run was 8388608.
==12287== 
==12287== HEAP SUMMARY:
==12287==     in use at exit: 0 bytes in 0 blocks
==12287==   total heap usage: 0 allocs, 0 frees, 0 bytes allocated
==12287== 
==12287== All heap blocks were freed -- no leaks are possible
==12287== 
==12287== For counts of detected and suppressed errors, rerun with: -v
==12287== ERROR SUMMARY: 1 errors from 1 contexts (suppressed: 0 from 0)
Segmentation fault (core dumped)
```

**Comment by artemisfowl** [comment]
Issue has been fixed. Next commit should push in the code fix.

```c
diff --git a/src/main.c b/src/main.c
index fad9727..9cd848b 100644
--- a/src/main.c
+++ b/src/main.c
@@ -12,7 +12,12 @@ int main(int argc, char *argv[])
        if (argc < MIN_ARGS || argc > MAX_ARGS) {
                printf("Error in number of arguments\n");
                p_display_usage();
-               exit(0);
+               exit(EXIT_SUCCESS);
+       } else if ((argc == MIN_ARGS && strcmp(argv[MIN_ARGS - 1], "-t") == 0)
+                       || (argc == MIN_ARGS + 1)) {
+               printf("Expected project type and project name\n");
+               p_display_usage();
+               exit(EXIT_SUCCESS);
        }
```

**Comment by artemisfowl** [comment]
Issue resolution addressed by 0d8a381a11d70f32fa9c3681dcc4fb85da0f4f86
